### PR TITLE
feat: video listing page added 

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Live preview 👍  : https://velvety-cranachan-b64fc8.netlify.app/movies 

files to Check 👍 : 
  
src/UI/Pages/VideoListingPage/VideoListingPage.jsx
src/UI/Pages/VideoListingPage/videolistingpage.css
src/components/Video-listing/VideoListing.jsx
 src/components/Video-listing/videolisting.css
src/utils/movies-data-fetcher.js

### CODE REVIEW

File: src/components/Video-listing/VideoListing.jsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  };
+
+  const removeToWatchLater = async (video, videoid) => {
+    const data = await deletewatchLater(videoid);
+    if (data.status === 200) {
+      watchLaterDispatch({ type: "REMOVE_FROM_WATCH_LATER", payload: video });
+    }
+  };
+
+  console.table(watchLaterVideos);
```
Comment (yaswanthmyneni) [inline src/components/Video-listing/VideoListing.jsx:42]: changes: remove console

File: src/components/Video-listing/VideoListing.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                  }}
+                  title="Remove From Watch Later"
+                >
+                  playlist_add_check_circle
+                </span>
+              ) : (
+                <span
+                  className="material-icons icon"
+                  onClick={() => {
+                    addToWatchLater(video);
```
Comment (yaswanthmyneni) [inline src/components/Video-listing/VideoListing.jsx:71]: suggestion: move this to a new component and call it here

File: src/components/Video-listing/videolisting.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+   background: rgba(255, 255, 255, 0.24);
+}
+
+.movie-card{
+    background: var(--bg-card);
+    border: 1px solid rgba(0,0,0,0.1);
+}
+
+.video-card{
+    background: #282b30;
```
Comment (yaswanthmyneni) [inline src/components/Video-listing/videolisting.css:18]: changes: use css variables for colors everywhere

File: src/components/Video-listing/VideoListing.jsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+
+  
+  const addVideoToHistory = async(movieDetail)=>{
+    const data = await historyVideo(movieDetail);
+    if(data.status === 201){
+     historyDispatch({type:'ADD_TO_HISTORY', payload:movieDetail})
+
+    }
+    console.log(data);
```
Comment (yaswanthmyneni) [inline src/components/Video-listing/VideoListing.jsx:24]: changes: here also remove console

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes

File: src/UI/Pages/VideoListingPage/VideoListingPage.jsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import "./videolistingpage.css";
+import { Navbar } from "../../../components/";
+
+import VideoListing from "../../../components/Video-listing/VideoListing";
```
Comment (prath-dev) [inline src/UI/Pages/VideoListingPage/VideoListingPage.jsx:5]: use barrel import export , on 3 line you are importing component  name export but here you are not using it . better to add these component in one index file of root of component section file and import it component from there

**Review by prath-dev — APPROVED** [review]

**Pull request merged**